### PR TITLE
New Git flow Cheat Sheet

### PR DESCRIPTION
**Pull request by tricinel**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [x] New Instant Answer
    - [x] Cheat Sheets: **`New Git flow Cheat Sheet`**

###### Description of changes

What changes does this PR introduce? 
Adding a cheat sheet for git flow which would help git users have an easier access to the basic git flow commands when working with features, releases and hotfixes.

Screenshot
git-flow-cheat-sheet-screenshot

---

Instant Answer Page: https://duck.co/ia/view/git_flow_cheat_sheet

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @tricinel

### CODE REVIEW

**Comment by daxtheduck** [comment]
@tricinel Thanks for the pull request! Unfortunately, we're only accepting contributions and improvements to a limited portion of the search space.
            
The staff and community are currently focused on making DuckDuckGo the [best search engine for programmers](https://forum.duckduckhack.com/t/duckduckhack-programming-mission-overview/53). To ensure we're making as much progress as possible, **we are only accepting Instant Answers and updates related to the programming mission**.
    
If this is not related to the Programming Mission, it will likely be closed but you can:
    
1. Help us prevent further confusion by telling us where you found out about DuckDuckHack or where we could've alerted you earlier.
2. Keep this code handy! We'll be expanding into other search topics in the future!
3. Join us at our new home: https://forum.duckduckhack.com/

If this is related to the Programming Mission, it will be reviewed by staff and the community as soon as possible!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/git-flow.json [hunk 1]
```diff
@@ -0,0 +1,79 @@
+{
+    "id": "git_flow_cheat_sheet",
+    "name": "Git flow",
+    "description": "A cheat sheet for Git flow",
+    
+    "metadata": {
+        "sourceName": "Cheatography",
+        "sourceUrl": "https://www.cheatography.com/vmalkani/cheat-sheets/git-flow"
```
Comment (akanshgulati) [inline share/goodie/cheat_sheets/json/git-flow.json]: @tricinel http://danielkummer.github.io/git-flow-cheatsheet/ check this one. It has better relevance with the sections you have added below.

**Comment by tricinel** [comment]
@akanshgulati Good points. The support branches are still experimental, AFAIK, that's why I didn't add that section in. Should I? And maybe just put in a flag in the section name, like "Working on support branches (EXPERIMENTAL)"?

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=git flow cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/git_flow_cheat_sheet_mobile.png?fXDoHtObbo)

**Comment by akanshgulati** [comment]
Thanks @tricinel for creating such a useful cheatsheet. You're absolutely right that git flow on support branches are still in experimental state. I suggest you add a "experimental" in brackets and add those commands too because these commands are even mentioned on official documentation. Let's add that and release this cheatsheet :)

**Comment by tricinel** [comment]
@akanshgulati added. Thanks! :)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=git flow cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/git_flow_cheat_sheet_mobile.png?WuMo77Bvwh)

**Pull request merged**